src/Servers/Kestrel/Core/src/Internal/HttpMultiplexedConnectionContext.cs

```diff
@@ -5,12 +5,15 @@
 using System.Net;
 using Microsoft.AspNetCore.Connections;
 using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3;
 using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure;
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal;
 
 internal sealed class HttpMultiplexedConnectionContext : BaseHttpConnectionContext
 {
+    internal IHttp3StreamLifetimeHandler _streamLifetimeHandler;
```

Comment:
Fields should always be private. Make it a property if you want it to be internal/public.